spec/services/stats/fee_scheme_usage_generator_spec.rb

```diff
@@ -59,10 +55,12 @@ def amend_claim(claim, case_type_id, case_type_name)
         'AGFS 13',
         'AGFS 14',
         'AGFS 15',
+        'AGFS 16',
```

Comment:
It might be better not to include the new fee scheme at this point. If you remove this line and line 103, you can treat this PR as a generic improvement to the tests. You could then add a new commit to your fee scheme PR to add fee scheme 16 to these tests. That way you don't need to create the dummy fee scheme on line 103, and all the changes relating to fee scheme 16 are encapsulated in the other PR.

Reply:
Good point, all the other tests still work without AGFS 16, they just now can have it appear without breaking. I'll remove these bits, make the other changes and update the PR in a moment